### DISCUSSION TITLE
Be able to attach issues and merge requests to multiple milestones

### DISCUSSION THREAD

**Discussion by MikeTheSnowman** [community/community#6804] [Projects and Issues]
Literally reposting an enhancement request from the old Github repo:
```
It would be useful to be able to attach issues and merge requests to multiple milestones.

A similar issue with discussions has also been created for gitlab (see here).

Anyone else thinks it would be useful?
```

Original enhancement request was from [here](https://github.com/isaacs/github/issues/441).

**Comment by michellemerrill** [comment]
Noting the 146 upvotes in [original request](https://github.com/isaacs/github/issues/441)

**Comment by darronschall** [comment]
I think this is a duplicate of an earlier discussion (with more activity): https://github.com/orgs/community/discussions/4566
